tests/manifests/special-cases/ppnd/usde-config.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
   labels:
     numaplane.numaproj.io/config: usde-config
 data:
-  defaultUpgradeStrategy: "pause-and-drain"
```

Comment:
is it possible that we can remove these entire files now? is there a difference with what gets deployed by default?